### ISSUE TITLE
Please add a BIG WARNING to not use win.meteor.com

### ISSUE THREAD

**Issue by dandv** [sdarnell/meteor-win#67]
Now that the [official Windows preview](https://github.com/meteor/meteor/wiki/Preview-of-Meteor-on-Windows) has been out for more than a month, it would be high time to warn about that on win.meteor.com.

There are still instructions out there that have people install from win.meteor.com, to their demise. E.g. [this guy on StackOverflow](https://stackoverflow.com/questions/28266117/cannot-add-package-iron-router-for-meteor-on-windows-8-1-pro).

**Comment by steph643** [comment]
I have to disagree. Unlike the methods described on win.meteor.com, the _official Windows preview_ still cannot be used in a professional environment.
To me, the main pending problem is the performance issue. Hope this will be fixed soon.

**Comment by dandv** [comment]
[The performance issue is being worked on at the very moment](https://github.com/meteor/windows-preview/issues/7) - down to 7 seconds per refresh.

The new Windows preview _works_ with Meteor 1.0+. This one (and kudoz to Steve's and Tom's work), no longer does, which trips up users.

**Comment by stubailo** [comment]
I think we're in the middle of a weird transitional period - hopefully in a few weeks we can say the official port is the best way to use Meteor on Windows for sure, but right now there are tradeoffs.

**Comment by Slava** [comment]
I would wait until we release.

**Comment by dandv** [comment]
How about a banner saying "If you have problems installing packages, then install the official preview". ? Or something that points to the official preview at all. Some people don't even know it exists.

**Comment by stubailo** [comment]
I think we would like most people to have a good first impression of the official port, rather than coming away with the thought that meteor is slow or buggy. We just need a little more time to finish up and then we will tell everyone about it.

**Comment by dandv** [comment]
That makes sense. The problem is people who land at win.meteor.com apparently can't even install packages (see [this](https://stackoverflow.com/questions/28266117/cannot-add-package-iron-router-for-meteor-on-windows-8-1-pro#comment44966015_28267002).

If updating win.meteor.com with an alternative Windows port === tell everyone about it, then by all means, that should not be done. Otherwise, I'm not sure I see a point in having a knowingly broken Windows port (win.meteor.com) without a link to a Windows port that does work, albeit not perfectly. 

BTW, the [README](https://github.com/meteor/windows-preview/blob/master/README.md) should be updated. There was lots of progress made since the first Windows preview release, and warning the user about the slow build is better than the users being surprised.

**Comment by sdarnell** [comment]
I have updated the text to hopefully reflect the current state of play at: 

However, for win.meteor.com you really need @gabrielhpugliese to do the update as I'm not able to publish the app.

Personally, I think a 'BIG WARNING' isn't appropriate, but a pointer allowing people to find out more seems reasonable. Then when the official port is ready, we win.meteor.com can probably be retired.
